README.md

```diff
@@ -411,65 +440,585 @@ Four actions are available for you to hook onto through the tooltip/popover life
+
+Asserts that a tooltip or popover is visible.
+
+This helper is usually used in conjunction with [triggerTooltipTargetEvent()](#triggerTooltipTargetEvent) to assert that a particular user interaction shows a tooltip to the user.
+
+For example:
+
+```js
+import {
+  assertTooltipVisible,
+  triggerTooltipTargetEvent,
+} from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`{{tooltip-on-element}}`);
+
+  triggerTooltipTargetEvent($(this), 'mouseenter');
+
+  /* Asserts that the tooltip is shown when the user hovers over the target, which is this test's element */
+
+  assertTooltipVisible(assert);
+});
+```
+
+You may use this helper with a variety of different user interactions. Here's an example that asserts that a tooltip is shown when the user focusses on an input:
+
+```js
+import {
+  assertTooltipVisible,
+  triggerTooltipTargetEvent,
+} from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`
+    <input id="url-input">
+    {{tooltip-on-element target='url-input'}}
+  `);
+
+  triggerTooltipTargetEvent($('#url-input'), 'focus');
+
+  /* Asserts that the tooltip is made visible when the user focuses on the input */
+
+  assertTooltipVisible(assert);
+});
+```
+
+This does not assert that the tooltip or popover is rendered in the DOM (regardless of visibility to the user) - use [assertTooltipRendered()](#assertTooltipRendered) for that.
+
+The [options hash](#test-helper-options) accepts:
+
+- [`selector`](#test-helper-option-selector)
+
+#### assertTooltipNotVisible()
+
+Asserts that a tooltip or popover is not visible.
+
+This helper is usually used in conjunction with [triggerTooltipTargetEvent()](#triggerTooltipTargetEvent) to assert that a particular user interaction hides a tooltip to the user.
+
+For example:
+
+```js
+import {
+  assertTooltipNotVisible,
+  assertTooltipVisible,
+  triggerTooltipTargetEvent,
+} from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`{{tooltip-on-element}}`);
+
+  /* Hover over the target to show the tooltip... */
+
+  triggerTooltipTargetEvent($(this), 'mouseenter');
+
+  assertTooltipVisible(assert);
+
+  /* Stop hovering over the target in order to hide the tooltip... */
+
+  triggerTooltipTargetEvent($(this), 'mouseleave');
+
+  assertTooltipNotVisible(assert);
+
+});
+```
+
+This helper is also used to assert that a tooltip is not visible even if it's been rendered in the DOM when this addon's [enableLazyRendering option](#enable-lazy-rendering) is enabled.
+
+For example:
+
+```js
+import {
+  assertTooltipNotVisible,
+  assertTooltipRendered,
+  triggerTooltipTargetEvent,
+} from 'appname/tests/helpers/ember-tooltips';
+
+test('Example test', function(assert) {
+
+  this.render(hbs`
+    {{tooltip-on-element
+      enableLazyRendering=true
+      event='click'
+    }}
+  `);
+
+  triggerTooltipTargetEvent($(this), 'mouseenter');
```

Comment:
a `mouseenter` event will not render a tooltip with a `click` event. The `assertTooltipRendered` will throw an error (we should probably fix that)